src/Raven.Server/Integrations/PostgreSQL/Handlers/Processors/AbstractPostgreSqlIntegrationHandlerProcessor.cs

```diff
@@ -24,10 +24,15 @@ public static void AssertCanUsePostgreSqlIntegration(TRequestHandler requestHand
 
         if (requestHandler.ServerStore.LicenseManager.CanUsePostgreSqlIntegration(withNotification: true))
         {
-            requestHandler.ServerStore.FeatureGuardian.Assert(Feature.PostgreSql, () =>
-                $"You have enabled the PostgreSQL integration via '{RavenConfiguration.GetKey(x => x.Integrations.PostgreSql.Enabled)}' configuration but " +
-                "this is an experimental feature and the server does not support experimental features. " +
-                $"Please enable experimental features by changing '{RavenConfiguration.GetKey(x => x.Core.FeaturesAvailability)}' configuration value to '{nameof(FeaturesAvailability.Experimental)}'.");
+            // if Postgres is enabled in config, we need to alert it's an experimental feature
+            if (requestHandler.ServerStore.Configuration.Integrations.PostgreSql.Enabled)
```

Comment:
shouldn't the `if` statements be swapped? the first one has `withNotification: true`

Reply:
I'm not sure it'd fit the flow here.
This method primarily checks the license. We just make extra effort to notify about experimental nature of the Postgre feature (only if If it's licensed and enabled).